code/game/machinery/kitchen/gibber.dm

```diff
@@ -199,21 +200,19 @@
 			slab_type = critter.meat_type
 	else if(istype(src.occupant,/mob/living/carbon/human))
 		var/mob/living/carbon/human/H = occupant
-		slab_name = src.occupant.real_name
-		slab_type = H.species.meat_type
+		slab_type = H.isSynthetic() ? /obj/item/stack/material/steel : H.species.meat_type

```

Comment:
Shouldn't meat_type on IPCs just be changed?

Reply:
Not only ipcs could be synthetics.

Reply:
They're the only synthetic that can reach this line, there's an ishuman statement gating this.

Reply:
I would prefer we keep this, in case we have other human types, that are not ipcs, that could also be synthetics.

Reply:
Those should implement meat_type correctly too.